### DISCUSSION TITLE
scatter plot

### DISCUSSION THREAD

**Discussion by dinaf39** [teamalgoritma/community#664] [Q&A] [answered]
![image](https://github.com/teamalgoritma/community/assets/130343206/37db0d2c-64a2-41fc-b49c-9a47a1d87599)
kenapa unsupported ya?

**Comment by nadyadtm** [comment]
Dear Bu @dinaf39 

Apakah bisa untuk di cek tipe data dari kolom 'App'nya? Jika tipe datanya masih berupa object atau string maka perlu diubah dulu dengan cara menggunakan `playstore["App"].astype("int")`

Semoga membantu.

**Reply by dinaf39** [reply]
![image](https://github.com/teamalgoritma/community/assets/130343206/3b081fd7-4a25-41a9-984c-830327275eca)
saya ganti ke int64 error mb, kl APP karena data nya seperti ini
![image](https://github.com/teamalgoritma/community/assets/130343206/066386a5-9bb2-4325-8ae3-86b44d523e2b)
jadi error mb 
![image](https://github.com/teamalgoritma/community/assets/130343206/bd29bcd2-3a55-4a8b-99a1-aea3dbb2b9b5)

**Comment by nadyadtm** [comment]
Baik bu, karena untuk bagian Area memerlukan kolom numerik jadi tidak bisa menggunakan kolom Apps. Jadi silakan untuk mencoba dengan menggunakan kolom numerik yang sudah tersedia atau sudah di preprocessing. Semoga membantu.

**Comment by nadyadtm** [comment, marked as answer]
Halo @dinaf39 , untuk menjaga agar forum diskusi berjalan kondusif, karena tidak ada respon lanjutan setelah lebih dari 7 hari dari respon terakhir, kami izin menandai pertanyaan sudah terjawab dan menutup diskusi ini. Semoga dapat dipahami dan terima kasih.
